.github/workflows/lint.yml

```diff
@@ -10,7 +10,7 @@
 name: Lint all codez
 on:
   push:
-    tags: 'release-*'
+    tags: 'v2**'
```

Comment:
why not simply `v**` if going for simple, or something more like `v\d{2}\.\d{2}.\d{1,3}` and specify the full regex? the inbetween regex is odd.

Reply:
you'll need to scape the dots as well. via `\.`

Reply:
Was just being consistent with all of the other actions https://github.com/vectorizedio/redpanda/blob/dev/.github/workflows/rpk-build.yml#L12-L14

I think we went with the simpler one to accomodate the (future) use of `-beta` etc

Reply:
Ok. Fair enough